### PR TITLE
czech translation

### PR DESCRIPTION
**Pull request by Amereyeu**

### CODE REVIEW

**Comment by Abestanis** [comment]
Hey, thank you for these translations!
I'm a unfortunately short on time today and in the coming week, but I will review this the week after, sorry for the delay.

In the meantime, out automatic tooling has picked up some problems with the new translations (you can see it [here](https://github.com/gyunaev/birdtray/actions/runs/4526314210/jobs/7971428458?pr=543)), please have a look at them (ignore the one warning in the German translation).

File: src/translations/installer_zh_cn.ts [hunk 1]
```diff
@@ -334,5 +334,9 @@ Visual C++ Runtime ${ARCH}
         <source>Greek</source>
         <translation>希腊语</translation>
     </message>
+    <message>
+        <source>Czech</source>
+        <translation>Lang_cs</translation>
```
Comment (Abestanis) [inline src/translations/installer_zh_cn.ts:339]:
```suggestion
        <translation>捷克语</translation>
```

File: src/translations/main_cs.ts [hunk 2]
```diff
@@ -0,0 +1,855 @@
...
+    </message>
+    <message>
+        <source>Support non-NETWM compliant WMs</source>
+        <translation>Podporovat WM, které nejsou kompatibilní s NETWM</translation>
+    </message>
+    <message>
+        <source>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;If your desktop manager is not fully NETWM compliant, you may need to check this checkbox so it can detect Thunderbird window and is able to minimise and hide it.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</source>
+        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokud váš správce plochy není plně kompatibilní s NETWM, možná budete muset zaškrtnout toto políčko, aby mohl detekovat okno Thunderbirdu a mohl jej minimalizovat a skrýt.&lt;/p&gt;&lt; /body&gt;&lt;/html&gt;</translation>
```
Comment (Abestanis) [inline src/translations/main_cs.ts:448]:
```suggestion
        <translation>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Pokud váš správce plochy není plně kompatibilní s NETWM, možná budete muset zaškrtnout toto políčko, aby mohl detekovat okno Thunderbirdu a mohl jej minimalizovat a skrýt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</translation>
```

File: src/translations/main_cs.ts [hunk 3]
```diff
@@ -0,0 +1,855 @@
...
+        <translation>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
+&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
+p, li { white-space: pre-wrap; }
+&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:&apos;Noto Sans&apos;; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt; font-weight:600;&quot;&gt;Verze Birdtray [VERSION] zkompilovaná [DATE] pomocí Qt [QT_VERSION].&lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:503]:
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 4]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Autorská práva (C) 2018 George Yunaev, &lt;/span&gt;&lt;a href=&quot;mailto:gyunaev@ulduzsoft.com&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration : underline; color:#2980b9;&quot;&gt;gyunaev@ulduzsoft.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:504]:
```suggestion
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Autorská práva (C) 2018 George Yunaev, &lt;/span&gt;&lt;a href=&quot;mailto:gyunaev@ulduzsoft.com&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;gyunaev@ulduzsoft.com&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 5]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:505]:
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 6]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Birdtray je SOFTWARE ZDARMA, který je licencován pod General Public License v3. Abychom to ještě upřesnili, můžete jej použít pro jakýkoli účel, včetně komerčního, aniž byste cokoli platili. &lt;/span&gt;&lt;/p&gt;
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:507]:
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 7]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Chcete-li požádat o pomoc, funkci nebo nahlásit chybu, použijte &lt;/span&gt;&lt;a href=&quot;https://github.com/gyunaev/birdtray&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;Stránka projektu Github&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size: 8pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:508]:
```suggestion
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Chcete-li požádat o pomoc, funkci nebo nahlásit chybu, použijte &lt;/span&gt;&lt;a href=&quot;https://github.com/gyunaev/birdtray&quot;&gt;&lt;span style=&quot; font-family:&apos;MS Shell Dlg 2&apos;; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;Stránka projektu Github&lt;/span&gt;&lt;/a&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;.&lt;/span&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 8]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:509]:
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 9]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Pokud chcete ocenit moji práci na Birdtray, který se vyvíjí v mém volném čase, můžete to udělat zde: &lt;/span&gt;&lt;a href=&quot;https://paypal. me/ulduzsoft&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;https://paypal.me/ulduzsoft&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:510]:
```suggestion
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style= &quot; font-size:8pt;&quot;&gt;Pokud chcete ocenit moji práci na Birdtray, který se vyvíjí v mém volném čase, můžete to udělat zde: &lt;/span&gt;&lt;a href=&quot;https://paypal.me/ulduzsoft&quot;&gt;&lt;span style=&quot; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;https://paypal.me/ulduzsoft&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 10]
```diff
@@ -0,0 +1,855 @@
...
+&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent :0px; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```
Comment (Abestanis) [inline src/translations/main_cs.ts:511]:
```suggestion
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8pt; text-decoration: underline; color:#2980b9;&quot;&gt;&lt;br /&gt;&lt;/p&gt;
```

File: src/translations/main_cs.ts [hunk 11]
```diff
@@ -0,0 +1,855 @@
...
+        <source>Download</source>
+        <translation>Stáhnout</translation>
+    </message>
+    <message>
+        <source>Ignore this version</source>
+        <translation>Ignorovat tuto verzi</translation>
+    </message>
+    <message>
+        <source>ca. %1 Mb</source>
+        <translation>cca. %1 Mb</translation>
```
Comment (Abestanis) [inline src/translations/main_cs.ts:821]: Is the double `c` in `cca.` intended, or is that a typo?

**Review by Abestanis — CHANGES_REQUESTED** [review]
Turns out I had some time to review this, I have left some suggestions and some requests for you. Once these are resolved, I will be happy to merge this.

File: src/translations/installer_cs.ts [hunk 12]
```diff
@@ -0,0 +1,293 @@
...
+<context>
+    <name>WindowsInstaller</name>
+    <message>
+        <source>__LANGUAGE_NAME__</source>
+        <translatorcomment>This must be the name of one of the NSIS translation files at https://sourceforge.net/p/nsis/code/HEAD/tree/NSIS/trunk/Contrib/Language%20files/ without the file extension.</translatorcomment>
+        <translation>Czech</translation>
+    </message>
+    <message>
+        <source>A free system tray notification for new mail for Thunderbird.</source>
+        <translation>BirdtraySectionDescription</translation>
```
[the remark this answers is not in the document]
Comment (Amereyeu) [reply src/translations/installer_cs.ts:13]: will do it today

**Review by Abestanis — CHANGES_REQUESTED** [review]
Nice, thank you very much for the installer translations.

I have left some suggestions regarding some technical parts of the translations as well as a question for you regarding one of the translations. You can see the suggestions on GitHub and accept them from there. If you tell me, I could also do that for you. Most of them are just removing of whitespaces in places where they can cause problems.

Other than the suggestions and the one question, this looks really good, thank you. 👍

**Pull request merged**